ui/package.json

```diff
@@ -182,6 +182,9 @@
     "tslint-microsoft-contrib": "^6.0.0",
     "typescript": ">=3.2.1 <3.3.0",
     "wd": "^1.12.1",
-    "webpack-dev-server": "3.1.11"
+    "webpack-dev-server": "^3.1.15"
+  },
+  "resolutions": {
```

Comment:
This is a bit sketchy, so please LMK if there are bad implications here.

This is necessary because Angular 7's `angular-devkit/build-angular@~0.13.10` requires the exact version `3.1.14`, which transitively pulls in a vulnerable yargs-parser.  This is the latest version of Angular 7, so this is the best we can do without upgrading Angular.  (an attempt at that is #4997)

Reply:
This seems OK since it's a dev dependency, and this workaround can be removed along with Angular.

Reply:
Great.  We can probably resolve a few more of the blocked-by-angular vulns in this way.